spec/support/coronavirus_feature_steps.rb

```diff
@@ -174,6 +174,79 @@ def when_i_visit_a_coronavirus_page
   visit "/coronavirus/landing"
 end
 
+### Reordering sections spec ##
+
+def when_i_visit_the_reorder_page
+  visit "/coronavirus/landing/reorder"
+end
+
+def set_up_basic_sub_sections
+  c = FactoryBot.create(:coronavirus_page, :landing)
+  FactoryBot.create(:sub_section,
+                    coronavirus_page_id: c.id,
+                    position: 0,
+                    title: "I am first",
+                    content: "###title\n[label](/url)")
+  FactoryBot.create(:sub_section,
+                    coronavirus_page_id: c.id,
+                    position: 1,
+                    title: "I am second",
+                    content: "###title\n[label](/url)")
+  path = Rails.root.join + "spec/fixtures/simple_coronavirus_page.yml"
+  github_yaml_content = File.read(path)
+  stub_request(:get, c.raw_content_url)
+    .to_return(status: 200, body: github_yaml_content)
+end
+
+def i_see_subsection_one_in_position_one
+  e = find("#step-0").find(".step-by-step-reorder__step-title")
```

Comment:
Again - a descriptive variable name would be preferable to a single letter one on a multi-line statement.